### PR TITLE
Intelligent apps - load balance - ACA

### PR DESCRIPTION
**Pull request by diberry**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/azure/ai/get-started-app-chat-scaling-with-azure-container-apps.md](https://github.com/dotnet/docs/blob/473c7ff8c7fafe0a72cb943f9339e248b665626c/docs/azure/ai/get-started-app-chat-scaling-with-azure-container-apps.md) | [CustomerIntent: As a .NET developer new to Azure OpenAI, I want to scale my OpenAI capacity to avoid rate limit errors.](https://review.learn.microsoft.com/en-us/dotnet/azure/ai/get-started-app-chat-scaling-with-azure-container-apps?branch=pr-en-us-39821) |

### CODE REVIEW

**Comment by diberry** [comment]
Ready for review @alexwolfmsft and @CamSoper.

**Review by CamSoper — APPROVED** [review]
LGTM!

**Comment by CamSoper** [comment]
@diberry Did you want me to merge or would you like to wait for @alexwolfmsft's review too?

**Comment by diberry** [comment]
@CamSoper Let's wait for @alexwolfmsft.

File: docs/azure/ai/get-started-app-chat-scaling-with-azure-container-apps.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+description: Learn how to add load balancing to your application to extend the chat app beyond the Azure OpenAI token and model quota limits.
+ms.date: 03/06/2024
+ms.topic: get-started
+ms.custom: devx-track-java, devx-track-java-ai
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want to scale my OpenAI capacity to avoid rate limit errors.
+---
+
+# Scale Azure OpenAI for .NET chat using RAG with Azure Container Apps
+
+[!INCLUDE [aca-load-balancer-intro](~/azure-dev-docs-pr/articles/intro/includes//scaling-load-balancer-introduction-azure-container-apps.md)]
```
Comment (alexwolfmsft) [inline docs/azure/ai/get-started-app-chat-scaling-with-azure-container-apps.md:12]: The file isn't in the PR, but can we update the include to link to "the chat app" - if someone lands on this page directly they might not know what that's referring to.

**Review by alexwolfmsft — APPROVED** [review]

File: docs/azure/ai/get-started-app-chat-scaling-with-azure-container-apps.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* Azure subscription.  [Create one for free](https://azure.microsoft.com/free/ai-services?azure-portal=true)
+* Access granted to Azure OpenAI in the desired Azure subscription.
+
+    Currently, access to this service is granted only by application. You should [apply for access](https://aka.ms/oai/access) to Azure OpenAI.
+
+* [Dev containers](https://containers.dev/) are available for both samples, with all dependencies required to complete this article. You can run the dev containers in GitHub Codespaces (in a browser) or locally using Visual Studio Code.
+
+#### [Codespaces (recommended)](#tab/github-codespaces)
+
+* GitHub account
```
Comment (alexwolfmsft) [inline docs/azure/ai/get-started-app-chat-scaling-with-azure-container-apps.md:25]: This looks a bit odd to me as a single bullet by itself - should we change this to say something like "Only a GitHub account is required to use CodeSpaces"

**Pull request merged**